docs/install-docker-quickstart.md

```diff
@@ -0,0 +1,167 @@
+# Docker quickstart
+
+To start developing with Nakama, you’ll first need to install it on your development machine. It’s straightforward and takes just a few minutes. In this guide we’re going to focus on installing your Nakama development instance using Docker.
+
+!!! summary "Recommended Approach"
+    Docker is the quickest way to download and get started with Nakama for development purposes. For production settings, we recommend that you install Nakama as a binary to ensure all system resources are available to Nakama.
+
+There is a single, minimal Nakama image that contains all of the Nakama binary. Each binary can be run by specifying the binary as the command when running Docker. The basic format is:
+
+```shell fct_label="Shell"
+docker run heroiclabs/nakama <command> [options]
+```
+
+Installing Nakama using Docker is ideal for a few reasons, including:
+
+- you install to a pristine environment
+
+- you get everything you need in one go, including CockroachDB
+
+- you can take snapshots, re-install and remove Nakama without affecting your primary operating system.
+
+- It also means that the installation instructions are the same whether your development machine runs Windows, MacOS and Linux.
+
+## What is Docker?
+
+If you’re new to Docker, then here’s what you need to know: Docker is an open soure containerization tool that lets you create multiple distinct Linux environments, each separate from the other.
+
+In a Docker container you run a suite of tools to do a particular job; in this case we’ll have one container running Nakama and another running CockroachDB. You can think of Docker containers as lightweight virtual machines.
+
+- Follow this [guide](https://www.docker.com/community-edition), if you are trying to install Docker on Mac, Linux and Windows 10 Pro edition.
+
+- [Docker Toolbox](https://www.docker.com/products/docker-toolbox) is needed, if you are installing Docker on Windows 7, 8 or 10 Home (non-Pro) editions.
+
+- Use the [Docker Store](https://store.docker.com/search?offering=community&q=&type=edition) to find the right version of Docker Community Edition for your environment.
+
+## Connecting the Nakama client
+
+Once Nakama is running via Docker, use the following connection detail to configure your client to connect to the server:
+
+**Host**: `127.0.0.1` (or `localhost`)    
+**Port** : `7350`     
+**SSL**: `False`    
+**Server Key**: `defaultkey`     
+
+In the Unity client, you can create an `NClient` like this:
+```csharp fct_label="Unity"
+INClient client = new NClient.Builder("defaultkey")
+    .Host("127.0.0.1")
+    .Port(7350)
+    .SSL(false)
+    .Build();
+```
+
+## Running Without Docker-Compose
+
```

Comment:
Maybe clarify exactly what Docker-Compose is here?   you've listed the positive trade-off, but if it's a true quickstart guide for someone who is new to Docker then maybe a one liner on it?